readme.txt

```diff
@@ -4,6 +4,7 @@ Tags: shipping, stamps, usps, woocommerce, taxes, payment, dhl, labels
 Requires at least: 4.6
 Requires PHP: 5.3
 Tested up to: 6.0.2
+WC tested up to: 7.0.0
```

Comment:
I tried to dig into another plugin like WC Composite Product, it uses 2 digits for the `Tested up to` and `WC tested up to` version. Should we follow them?

Reply:
Just saw your conversation about this on [slack thread](https://progressus.slack.com/archives/C022B9ALCQ3/p1666101800610369?thread_ts=1665620842.299059&cid=C022B9ALCQ3). Then I think there is no need to change this into 2 digits.